config/tf_modules/aws-s3/bucket.tf

```diff
@@ -64,6 +66,27 @@ resource "aws_s3_bucket" "bucket" {
   }
 }
 
+resource "aws_cloudfront_origin_access_identity" "read" {
```

Comment:
Can you add a comment here: that it's fine adding this as it's just creating something akin to an IAM role.

Reply:
done